awx/main/models/schedules.py

```diff
@@ -207,23 +245,33 @@ def rrulestr(cls, rrule, fast_forward=True, **kwargs):
                 raise ValueError('A valid TZID must be provided (e.g., America/New_York)')
 
         # Fast forward is a way for us to limit the number of events in the rruleset
-        # If we are fastforwading and we don't have a count limited rule that is minutely or hourley
+        # If we are fast forwarding and we don't have a count limited rule that is minutely or hourly
         # We will modify the start date of the rule to last week to prevent a large number of entries
         if fast_forward:
             try:
                 # All rules in a ruleset will have the same dtstart value
-                #   so lets compare the first event to now to see if its > 7 days old
+                # so lets compare the first event to now to see if its > 30 days old
+                # we choose > 30 hours because if rrule has FREQ=HOURLY and INTERVAL=23, it will take
```

Comment:
Comment says "> 30 days old" and then "> 30 hours". This doesn't look intentional.

Reply:
oh thanks, fixed